bulk-user-setup/sample_test.csv

```diff
@@ -0,0 +1,6 @@
+email,firstName,lastName,roles
+rejum@tedek.by,Sophie,Owen,caseworker|caseworker-autotest1|caseworker-autotest1-private
+cepumo@gopwisu.tp,Matilda,Reynolds,caseworker|caseworker-autotest1
+ejeso@men.ci,Adele,Norris,caseworker|caseworker-autotest1|caseworker-autotest1-private
+fobip@fic.va,Alexander,Wagner,caseworker
+luazfi@nihisasi.mx,Eugenia,Hardy,caseworker
```

Comment:
maybe we could consider a "invite" flag column with Y or N, to distinguish the rows in a file for which the user was already successfully invited in a previous trial. And then, the script could just re-write the file with the updated flags upon successful/failed calls in a row.